packages/govern-subgraph/schema.graphql

```diff
@@ -16,28 +16,18 @@ type Govern @entity {
   id: ID!
   address: Bytes!
   metadata: Bytes
-  games: [OptimisticGame!] @derivedFrom(field: "executor")
-  executions: [Execution!]
+  correspondingEntries: [RegistryEntry!] @derivedFrom(field: "executor")
+  containers: [Container]!
   roles: [Role!]
 }
 
-type Execution @entity {
```

Comment:
I think we need to have some way to track the execution that a Govern instance does so we keep a record of the results of the given execution.

Reply:
We could probably just record these on the `data` field of the corresponding `ContainerEvent`?